vagrant/Vagrantfile

```diff
@@ -91,8 +120,11 @@ Vagrant.configure("2") do |config|
     export CCMAKE="ccmake -DCMAKE_PREFIX_PATH:PATH=$INSTALL_DIR/lib/cmake -DCMAKE_INSTALL_PREFIX:PATH=$INSTALL_DIR"
 
     git clone https://github.com/CCPPETMR/CCPPETMR_VM.git
-    bash CCPPETMR_VM/scripts/INSTALL_prerequisites_with_apt-get.sh
-    #bash CCPPETMR_VM/scripts/UPDATE.sh
+	cd CCPPETMR_VM
+	git pull
+	git checkout v0.9.2
+    bash $SRC_PATH/CCPPETMR_VM/scripts/INSTALL_prerequisites_with_apt-get.sh
+    bash $SRC_PATH/CCPPETMR_VM/scripts/UPDATE.sh
```

Comment:
next line needs to be
```
chown -R sirfuser:users /home/sirfuser/
```
Otherwise various files are still owned by root

Reply:
yes